content/docs/tutorials/understanding_metric_types.md

```diff
@@ -3,13 +3,15 @@ title: Understanding metric types
 sort_rank: 2
 ---
 
-# Types of metrics.
+# Understanding metric types
 
 Prometheus supports four types of metrics, they are
     - Counter
     - Gauge
     - Histogram
-    - Summary 
+    - Summary
```

Comment:
Even with the changes, this still doesn't seem to render as a proper markdown list (see deploy preview at https://deploy-preview-2251--prometheus-docs.netlify.app/docs/tutorials/understanding_metric_types/). Probably needs a space before the first item, and/or less indentation or something?